.github/workflows/main.yml

```diff
@@ -4,11 +4,14 @@ name: CI
 
 # Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the "main" branch
+  # # Triggers the workflow on push or pull request events but only for the "main" branch
```

Comment:
You have to revert this right before merging to main?

Reply:
Will do!